pkg/kubelet/dockertools/manager_test.go

```diff
@@ -115,6 +115,33 @@ func matchString(t *testing.T, pattern, str string) bool {
 	return match
 }
 
+func TestNewDockerVersion(t *testing.T) {
+	cases := []struct {
+		value string
+		out   string
+		err   bool
+	}{
+		{value: "1", err: true},
+		{value: "1.8", err: true},
```

Comment:
Semver is x.y.z, both of these are not semver.  Not aware of a Docker build in the wild that isn't base semver.